### DISCUSSION TITLE
Can i play on 2 computer?

### DISCUSSION THREAD

**Discussion by TccKaiCi** [HiamKaito/NUMBER_FINDING_GAME#9] [Q&A] [answered]
Can i play on 2 computer?

**Comment by TccKaiCi** [comment, marked as answer]
Yes. You can use VPS for a host/server and client for join.
I use **[radminVPN](https://www.radmin-vpn.com/)** for Server and Client, like a LAN.

**Reply by tienem3243** [reply]
I no sure that is legal. But may be you can try

**Reply by prohzgod** [reply]
no it didnt, it just can play on 1 computer because no one interested with it to play it

**Reply by TccKaiCi** [reply]
> no it didnt, it just can play on 1 computer because no one interested with it to play it

LMAO
